### PR TITLE
react-form-controls complete

### PR DESCRIPTION
**Pull request by luiderek**
userpass

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          type="text"
+          value={this.state.username}
+          onChange={this.handleUsernameChange}
+        />
+        <input
+          type="password"
+          value={this.state.password}
+          onChange={this.handlePasswordChange}
+        />
+        <button onClick={this.handleSubmit}>Sign Up</button>
```
Comment (TimelikeClosure) [inline react-form-controls/src/index.jsx:39]: Use form `submit` events instead of button `click` events.

File: react-form-controls/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        <input
+          type="text"
+          value={this.state.username}
+          onChange={this.handleUsernameChange}
+        />
+        <input
+          type="password"
+          value={this.state.password}
+          onChange={this.handlePasswordChange}
+        />
```
Comment (TimelikeClosure) [inline react-form-controls/src/index.jsx:38]: Each form control needs a `<label>` and a `name` attribute.

File: react-form-controls/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class RegristrationForm extends React.Component {
```
Comment (TimelikeClosure) [inline react-form-controls/src/index.jsx:4]: Your component name has a typo

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by luiderek** [comment]
image

File: react-form-controls/src/index.jsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          />
+        </label>
+        <label name="Password">
+          Password
+          <input
+            type="password"
+            value={this.state.password}
+            onChange={this.handlePasswordChange}
+          />
+        </label>
```
Comment (TimelikeClosure) [inline react-form-controls/src/index.jsx:46]: `name` attributes go on form controls, not their labels, otherwise they don't do anything.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: react-form-controls/src/index.jsx (hunk 4 above)
Comment (luiderek) [reply react-form-controls/src/index.jsx:46]: ah, shoulda picked up on VSC not autosuggesting. thanks!

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**